diff --git a/node_modules/ts-nkeys/lib/util.js b/node_modules/ts-nkeys/lib/util.js
index 8950a49..0cac641 100644
--- a/node_modules/ts-nkeys/lib/util.js
+++ b/node_modules/ts-nkeys/lib/util.js
@@ -50,12 +50,12 @@ function parseNodeVersion() {
 // is converted to ArrayBuffer. These older nodes
 // don't have the '.buffer' property.
 function toArrayBuffer() {
-    if (parseNodeVersion() < 8) {
-        return node6;
-    }
-    else {
+//  if (parseNodeVersion() < 8) {
+//      return node6;
+//  }
+//  else {
         return node8;
-    }
+//  }
 }
 exports.toArrayBuffer = toArrayBuffer;
 //# sourceMappingURL=util.js.map